src/lightning_lite/utilities/enums.py

```diff
@@ -0,0 +1,119 @@
+from __future__ import annotations
```

Comment:
GradClipAlgorithmType and AutoRestartBatchKeys are staying in PL, hence this is no clean file move.
No implementation changes to the moved functions here.